bitski/src/lib.rs

```diff
@@ -51,9 +51,10 @@ impl Bitski {
     }
 
     pub fn from_env() -> Result<Self, Error> {
-        let client_id = std::env::var("API_KEY").or_else(|_| std::env::var("CLIENT_ID"))?;
-        let credential_id = std::env::var("CREDENTIAL_ID");
-        let credential_secret = std::env::var("CREDENTIAL_SECRET");
+        let client_id =
+            std::env::var("BITSKI_CLIENT_ID").or_else(|_| std::env::var("BITSKI_CLIENT_ID"))?;
+        let credential_id = std::env::var("BITSKI_CREDENTIAL_ID");
+        let credential_secret = std::env::var("BITSKI_CREDENTIAL_SECRET");
         let scopes: Vec<String> = std::env::var("SCOPES")
```

Comment:
Should rename this one too

Reply:
Are you referring to SCOPES -> BITSKI_SCOPES?

Reply:
Yeah